packages/admin/admin/src/actionLog/header/ActionLogHeader.tsx

```diff
@@ -0,0 +1,110 @@
+import { Box, type ComponentsOverrides, type Theme, type Typography, useThemeProps } from "@mui/material";
+import { type ReactNode } from "react";
+import { FormattedMessage } from "react-intl";
+
+import { type ThemedComponentBaseProps } from "../../helpers/ThemedComponentBaseProps";
+import { DbTypeLabel, DbTypeValue, InfoContainer, Root, TitleContainer, TitleTypography, UuidLabel, UuidValue } from "./ActionLogHeader.sc";
+
+export type ActionLogHeaderClassKey =
+    | "root"
+    | "titleContainer"
+    | "infoContainer"
+    | "title"
+    | "uuidLabel"
+    | "uuidValue"
+    | "dbTypeLabel"
+    | "dbTypeValue";
+
+export interface ActionLogHeaderProps
+    extends ThemedComponentBaseProps<{
+        root: typeof Box;
+        titleContainer: typeof Box;
+        infoContainer: typeof Box;
+        title: typeof Typography;
+        uuidLabel: typeof Typography;
+        uuidValue: typeof Typography;
+        dbTypeLabel: typeof Typography;
+        dbTypeValue: typeof Typography;
+    }> {
+    action?: ReactNode;
+    dbTypes?: string[];
+    id: string;
+    title: ReactNode;
+}
+
+export const ActionLogHeader = (inProps: ActionLogHeaderProps) => {
+    const {
+        action,
+        dbTypes = [],
+        id,
+        title,
+        slotProps,
+        ...restProps
+    } = useThemeProps({
+        props: inProps,
+        name: "CometAdminActionLogHeader",
+    });
+
+    return (
+        <Root {...restProps} {...slotProps?.root}>
+            <TitleContainer {...slotProps?.titleContainer}>
+                <TitleTypography variant="h3" {...slotProps?.title}>
+                    {title}
+                </TitleTypography>
+            </TitleContainer>
+
+            <InfoContainer {...slotProps?.infoContainer}>
+                <Box display="flex" gap={2}>
```

Comment:
Technically this should be a slot and we shouldn't mix props-styling and slots-styling 🙂